server.js

```diff
@@ -30,6 +32,26 @@ const makeSgRequest = body =>
     body: body.toJSON()
   })
 
+// This converts {a:1, b:2} into 'a=1&b=2'
+const queryParams = obj =>
+  Object.keys(obj)
+    .map(key => [key, obj[key]]) // There is no Object.entries() in node 6
+    .map(([key, val]) => `${key}=${val}`)
+    .join('&')
+
+const wrikeMkFolder = name =>
+  fetch(process.env.WRIKE_URL, {
+    body: queryParams({
```

Comment:
You are right, in this moment we are using fixed data but the real data is the one coming from the user and it could be anything. But maybe not wrap the whole string, we don't want to encode the `=` and `&` in it.